src/BlazorWebView/src/Maui/Android/BlazorWebViewHandler.Android.cs

```diff
@@ -99,6 +99,12 @@ private void StartWebViewCoreIfPossible()
 				VirtualView.JSComponents,
 				hostPageRelativePath);
 
+			((BlazorWebView)VirtualView).NotifyBlazorWebViewInitializing(new BlazorWebViewInitializingEventArgs());
```

Comment:
There's no much point in us firing this event, but it would be weird if we didn't. I don't think there's anything a user can do for Android here as there is no context passed in.

Reply:
I definitely think we should raise the event consistently on every platform. The user might just have their own startup code they want to run "at that time" even if it's not related to the BlazorWebView implementation. Plus, in the future we might have something to pass in here.

Reply:
That sounds good, I'll leave it as is then so that we can add stuff in the future if needed.